go/tasks/pluginmachinery/internal/webapi/allocation_token.go

```diff
@@ -23,27 +23,37 @@ func newTokenAllocator(c clock.Clock) tokenAllocator {
 	}
 }
 
+type TokenDetails struct {
```

Comment:
what is this structure for? 
It seems it is associated with the PhaseInfo, if so, why can we not add it to PhaseInfo?

Reply:
it's specific to the webapi token allocator, wan't sure it was worth broadly making accessible

Reply:
no longe relevant